examples/mnit_moles/src/moles.nit

```diff
@@ -23,64 +23,81 @@ module moles is
 end
 
 import mnit
+import realtime
 
+import drawing
```

Comment:
The module is generated by the Makefile. This is what make the tests fails because it only compiles the Nit code and does not run make. If I were to move it to contrib, would it work? If I am not mistaken, Jenkins runs make on each contrib?